src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -382,9 +384,10 @@ MavlinkMissionManager::send_mission_item(uint8_t sysid, uint8_t compid, uint16_t
 		send_mission_ack(_transfer_partner_sysid, _transfer_partner_compid, MAV_MISSION_ERROR);
 
 		if (_filesystem_errcount++ < FILESYSTEM_ERRCOUNT_NOTIFY_LIMIT) {
-			_mavlink->send_statustext_critical("Mission storage: Unable to read from microSD\t");
-			events::send(events::ID("mavlink_mission_storage_read_failure"), events::Log::Error,
-				     "Mission: Unable to read from storage");
+			mavlink_log_critical(_mavlink->get_mavlink_log_pub(),
+					     "Mission storage: Unable to read from storage, result: %" PRId16 " %" PRId16 "\t", _mission_type, bytes_read);
+			events::send<uint8_t, int16_t>(events::ID("mavlink_mission_storage_read_failure"), events::Log::Error,
```

Comment:
The details can be in the description:
```
/** EVENT
 * @description Mission type: {1}. Number of bytes read: {2}
 */
```

Reply:
Thanks for the clarification! Today I learned https://github.com/PX4/PX4-Autopilot/pull/19655#discussion_r873343041 💡 I'm adjusting.

Reply:
Addressed in 2d08886236555cff5ec2e0e15cce1d42551dcd7b